llvm/lib/Support/APFloat.cpp

```diff
@@ -316,6 +362,8 @@ unsigned int APFloatBase::semanticsIntSizeInBits(const fltSemantics &semantics,
 
 bool APFloatBase::isRepresentableAsNormalIn(const fltSemantics &Src,
                                             const fltSemantics &Dst) {
+  // FIXME implement for DFP
```

Comment:
For this method, the fix is that it should return false if src and dst have different radixes.

Reply:
`isRepresentableAsNormalIn` isn't documented with a doxygen comment right now, but the comment above its definition says
> Returns true if any number described by \p Src can be precisely represented by a normal (not subnormal) value in \p Dst.

Since many numbers in base 2 can't be precisely represented by a base 10 with any precision (and vice versa), it's vacuously false if they have different radixes.

Reply:
I am not going to implement the fix for now. I don't think it fall into the next PR either.

Reply:
There is a FIXME comment. I think that is good enough for now.